### PR TITLE
Use embedded config in circleci testing

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -28,8 +28,14 @@ commands:
...
             # Will add emsdk version of node to PATH
-            source ~/emsdk-master/emsdk_env.sh
+            source ~/emsdk/emsdk_env.sh
             npm ci
+  set-config:
+    description: "set EM_CONFIG"
+    steps:
+      - run:
+          name: Set EM_CONFIG
+          command: echo "export EM_CONFIG=\"$HOME/emsdk/.emscripten\"" >> $BASH_ENV
```
Comment (kripken) [inline .circleci/config.yml:38]: why is this needed? isn't the default to use an embedded config, i.e., to put the config file exactly there?

Comment (sbc100) [reply .circleci/config.yml]: Updated.  Its needed because we are trying to test out version emscripten (which is not in the emsdk tree) against the emsdk config. 

The emscripten lives in the emsdk tree can find the emedded config, but the out-of-tree emscripten can't find it automatically.

An alternative would be to checkout emscripten inside the emsdk tree..   that might be cleaner but I've already done it this way now.
